artifactory/groups.go

```diff
@@ -54,11 +55,23 @@ func (s *GroupsService) GetAll() (*[]Group, *Response, error) {
 	return v, resp, err
 }
 
+// GetGroupRequest represents the possible inputs for a Get Group request.
+//
+// Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetGroupDetails
+type GetGroupRequest struct {
+	Name         *string
+	IncludeUsers *bool
+}
+
 // Get returns the provided group.
 //
 // Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetGroupDetails
-func (s *GroupsService) Get(group string) (*Group, *Response, error) {
-	u := fmt.Sprintf("/api/security/groups/%s", group)
+func (s *GroupsService) Get(groupRequest *GetGroupRequest) (*Group, *Response, error) {
+	u := fmt.Sprintf("/api/security/groups/%s", *groupRequest.Name)
```

Comment:
Thoughts on potentially using the struct accessor instead?

I know it's a bit nit-picky, but if possible, we like to encourage using them as they add a small bit of safety from panics due to nil pointer dereference errors.

To clarify, instead of this:

```go
u := fmt.Sprintf("/api/security/groups/%s", *groupRequest.Name)
```

what if we did this instead?

```go
u := fmt.Sprintf("/api/security/groups/%s", groupRequest.GetName())
```

Reply:
Makes sense. Happy to refactor this.